### PR TITLE
Update common-rangefinder-nooploop doc

### PR DESCRIPTION
**Pull request by daiyingjun**
Due to product iteration, updating manuals and product names

### CODE REVIEW

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
under/overlines must be same length as title text

File: common/source/docs/common-rangefinder-nooploop-tofsense-f.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
 .. _common-rangefinder-nooploop-tofsense-f.rst:
 
 ===================
-Nooploop TOFSense F
+Nooploop TOFSense F/FP/F2 Mini/F2/F2 P
```
Comment (Hwurzburg) [inline common/source/docs/common-rangefinder-nooploop-tofsense-f.rst:4]: fix under/overlines

File: common/source/docs/common-rangefinder-nooploop-tofsense-p.rst [hunk 2]
```diff
@@ -1,13 +1,13 @@
 .. _common-rangefinder-nooploop-tofsense-p:
 
 ===================
-Nooploop TOFSense P
+Nooploop TOFSense/-UART/S
```
Comment (Hwurzburg) [inline common/source/docs/common-rangefinder-nooploop-tofsense-p.rst:4]: under/overline

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-rangefinder-nooploop-tofsense-p.rst (hunk 2 above)
Comment (daiyingjun) [reply common/source/docs/common-rangefinder-nooploop-tofsense-p.rst:4]: Sorry, I didn't quite understand what you meant. Our product name is TOFSense-UART. Do you mean we need to change/- UART to/UART here?

Comment (Hwurzburg) [reply common/source/docs/common-rangefinder-nooploop-tofsense-p.rst:4]: no, the row of "=" above and below titles must be the same length as the text of the tiltle

Comment (daiyingjun) [reply common/source/docs/common-rangefinder-nooploop-tofsense-p.rst:4]: Okay, thank you. It's fixed

Comment (Hwurzburg) [reply common/source/docs/common-rangefinder-nooploop-tofsense-p.rst:4]: looks good...I squashed for you...will merge once CI passes again

**Pull request merged**